phys/module_cu_ntiedtke.F

```diff
@@ -694,7 +703,7 @@ subroutine cumastrn  &
      &     pssfc,    ldcum,                              &
      &     ktype,    kcbot,    kctop,    ptu,      pqu,&
      &     plu,      plude,    pmfu,     pmfd,     prain,&
-     &     pcte,     phhfl,    lndj,     zgeoh,   dx)
+     &     pcte,     phhfl,    lndj,     zgeoh,    dx)
```

Comment:
@kkeene44 can you remove the space before dx so that this won't show up as a change?

Reply:
@weiwangncar 
Done.